### PR TITLE
Bootstrap v5: Base & Utilities

### PR DESCRIPTION
**Pull request by jovyntls**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [x] Code maintenance
- [ ] Others, please explain:

Part of #1702; this PR focuses on updating the docs and bootstrap version, as well as some bootstrap utilities

**Overview of changes:**
- Updated the links in the docs to point to Bootstrap v5, except for Glyphicons which were deprecated from Bootstrap in v4
- Switch to Bootstrap v5.1.3 for
	- `bootstrap.min.css`
	- `bootstrap.min.css.map`
	- `bootstrap-utility.min.js` 
- Migrated the breaking changes on [utilities](https://getbootstrap.com/docs/5.0/migration/#utilities)
	- `.ml-*` and `.mr-*` -> `.ms-*` and `.me-*`
	- `.font-weight-*` -> `.fw-*`
	- `float-left` and `float-right` -> `float-start` and `float-end`
	- remove newly default underline for `<a>` elements

**Anything you'd like to highlight / discuss:**
- Bootstrap v5 also introduces quite a few utility class that may be useful, but I think that can be addressed in a separate issue for refactoring after the migration.
- The official javascript [file](https://getbootstrap.com/docs/5.1/getting-started/contents/#js-files) provided by Bootstrap is called `bootstrap.min.js`, but MarkBind uses the name `bootstrap-utility.min.js` (same file content) starting from [this commit](https://github.com/MarkBind/markbind/pull/465/commits/bbef2c1ab6243b949ae788b814b2617de89233e1). 
	- I couldn't find any reason why it was called `bootstrap-utility.min.js` instead of the default name. Should we rename it to `bootstrap.min.js`to make it clear that we are using the exact minified JS file provided by Bootstrap? Using a different name may suggest that we are using a different file or are using the file for a different purpose.

**Testing instructions:**
Most styles are still buggy (not migrated yet). 
For now, most of the above changes should work (margin utility classes, font weight utility classes, floats) unless they have been overwritten by another component/class.

**Proposed commit message: (wrap lines at 72 characters)**
```
Migrate utilities to Bootstrap V5
```

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No unrelated changes 

---

### CODE REVIEW

File: docs/userGuide/syntax/modals.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ header{{slot_info_trigger}} | `String` | `''` | Header of the Modal component. S
...
 ok-text | `String` | `''` | Text for the OK button.
 effect | `String` | `zoom` | Supports: `zoom`, `fade`.
-id | `String` | | Used by [Trigger](#trigger) to activate the Modal by id.large | `Boolean` | `false` | Creates a [large Modal](https://getbootstrap.com/docs/4.0/components/modal/#optional-sizes).
-small | `Boolean` | `false` | Creates a [small Modal](https://getbootstrap.com/docs/4.0/components/modal/#optional-sizes).
-large | `Boolean` | `false` | Creates a [large Modal](https://getbootstrap.com/docs/4.0/components/modal/#optional-sizes).
+id | `String` | | Used by [Trigger](#trigger) to activate the Modal by id.large | `Boolean` | `false` | Creates a [large Modal](https://getbootstrap.com/docs/5.1/components/modal/#optional-sizes).
+small | `Boolean` | `false` | Creates a [small Modal](https://getbootstrap.com/docs/5.1/components/modal/#optional-sizes).
+large | `Boolean` | `false` | Creates a [large Modal](https://getbootstrap.com/docs/5.1/components/modal/#optional-sizes).
```
Comment (jonahtanjz) [inline docs/userGuide/syntax/modals.md:50]: I think these should be removed since it is no longer using `bootstrap-vue`?

**Review by jonahtanjz — COMMENTED** [review]
Nice work @jovyntls! Overall looks good.

Missed out on a few here:

[Question.vue#77](https://github.com/jovyntls/markbind/blob/bootstrap5-migration/packages/vue-components/src/questions/Question.vue#L77) (`float-right`)

[packages/cli/test/functional/test_site/_markbind/layouts/default.md#16](https://github.com/jovyntls/markbind/blob/bootstrap5-migration/packages/cli/test/functional/test_site/_markbind/layouts/default.md?plain=1#L16) (`font-weight-bold`)

> I couldn't find any reason why it was called bootstrap-utility.min.js instead of the default name. Should we rename it to bootstrap.min.js to make it clear that we are using the exact minified JS file provided by Bootstrap? Using a different name may suggest that we are using a different file or are using the file for a different purpose.

I'm not entirely sure why it was renamed. I'm leaning towards using the original file name (`bootstrap.min.js`) as well so that it is clearer. Maybe @ang-zeyu can chip in on this?

File: docs/userGuide/syntax/modals.md (hunk 1 above)
Comment (jovyntls) [reply docs/userGuide/syntax/modals.md:50]: These should have been removed in #1873 — will merge the latest changes from master into this branch!

**Review by jonahtanjz — APPROVED** [review]
Thank @jovyntls 

LGTM 👍 

EDIT: I'll merge this in first. If we later on decide to rename the javascript file, can do it in another PR :)

**Pull request merged**